### PR TITLE
@types/hls.js : Fixing definition of Level interface

### PR DESCRIPTION
**Pull request by adripanico**
Please fill in this template.

- [DONE] Use a meaningful title for the pull request. Include the name of the package modified.
- [DONE] Test the change in your own code. (Compile and run.)
- [N/A] Add or edit tests to reflect the change. (Run with `npm test`.)
- [DONE] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [DONE] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [DONE] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [See comment below] Provide a URL to documentation or source code which provides context for the suggested changes.
- [N/A] Increase the version number in the header if appropriate.
- [N/A] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/hls.js/index.d.ts [hunk 1]
```diff
@@ -189,33 +190,61 @@ declare namespace Hls {
...
         name: string;
         /**
-         * used codecs
+         * array of unrecognized codecs
          */
-        codecs: string;
+        unkownCodecs: string[];
         /**
-         * video width
+         * level url. might contain sever items if failover/redundant streams are found in the manifest
```
Comment (jgainfort) [inline types/hls.js/index.d.ts]: typo: several

File: types/hls.js/index.d.ts [hunk 2]
```diff
@@ -189,33 +190,61 @@ declare namespace Hls {
      */
     interface Level {
         /**
-         * level url. might contain sever items if failover/redundant streams are found in the manifest
+         * attribute list
          */
-        url: string[];
+        attrs: Array<{[key: string]: string}>;
```
Comment (jgainfort) [inline types/hls.js/index.d.ts]: Personally I prefer `{[key: string]: string}[]` or even better to define an interface or a type
`interface AttrList { [key: string]: string }`

**Comment by jgainfort** [comment]
@adripanico Thank you for the PR!

File: types/hls.js/index.d.ts (hunk 2 above)
Comment (adripanico) [reply types/hls.js/index.d.ts]: I will use your second approach then. The first one raises a linter error.

**Review by jgainfort — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Comment by adripanico** [comment]
It isn't my first contribution, but thanks!

**Pull request merged**